iina/MPVController.swift

```
@@ -23,7 +23,7 @@ fileprivate let no_str = "no"
  "debug" - very noisy technical information
  "trace" - extremely noisy
  */
-fileprivate let MPVLogLevel = "warn"
+fileprivate let MPVLogLevel = "debug"
```

The `libmpv` event system uses a common design which has this critical requirement:
>  The internal event queue has a limited size (per client handle). If you don't empty the event queue quickly enough with mpv_wait_event(), it will overflow and silently discard further events. If this happens, making asynchronous requests will fail as well (with MPV_ERROR_EVENT_QUEUE_FULL).

IINA can recover from the loss of some types of mpv events, but certain types of mpv events are critical. If one of the critical mpv events is silently discarded IINA will experience severe malfunctions, for example a permanent hang that requires quitting and restarting IINA to clear.

As IINA's ability to include some mpv log messages in the IINA log file relies up the mpv `MPV_EVENT_LOG_MESSAGE` event, there is the danger that mpv will emit log messages at a rate that exceeds IINA's ability to empty the event queue before it overflows. For this reason IINA intentionally limits the mpv log messages IINA logs to `warn` level.

I have been planning on adding some comments to `MPVController` to point out this important concern when handling mpv events.